bootstrap/bungeecord/src/main/java/org/geysermc/geyser/platform/bungeecord/GeyserBungeeInjector.java

```diff
@@ -157,7 +205,10 @@ protected void initChannel(@NonNull Channel ch) throws Exception {
                     }
                 })
                 .childAttr(listener, listenerInfo)
-                .group(bossGroup, workerGroup)
+                .group(new MultiThreadIoEventLoopGroup(LocalIoHandler.newFactory()), wrapperGroup)
+                // Hardcoded to pooled allocator on BungeeCord
+                // https://github.com/SpigotMC/BungeeCord/blob/master/proxy%2Fsrc%2Fmain%2Fjava%2Fnet%2Fmd_5%2Fbungee%2Fnetty%2FPipelineUtils.java#L224
```

Comment:
Make this point to a static commit (like in getNativeHandlerFactory below) so additional code changes in Bungee can't move it around!